flake.nix

```diff
@@ -34,6 +40,34 @@
           src = pkgs.lib.cleanSource ./py/sigver;
           propagatedBuildInputs = pythonDependencies;
         };
+
+	sbsignPkg = sbsigntools.packages.${system}.default;
+	akvenginePkg = akvengine.packages.${system}.default;
+
+	signmeScript = pkgs.writeShellScriptBin "signme" ''
+
+	  set -euo pipefail
+
+	  tmpconf=$(mktemp)
+	  cat > "$tmpconf" <<EOF
+ openssl_conf = openssl_init
+
+ [openssl_init]
+ engines = engine_section
+
+ [engine_section]
+ akv = akv_section
+
+ [akv_section]
+ engine_id = akv
+ dynamic_path = ${akvenginePkg}/lib/engines-3/e_akv.so
+ init = 1
+EOF
+
+	  export OPENSSL_CONF="$tmpconf"
```

Comment:
No need to write this to a tmpfile, use `replaceVars` from nixpkgs to get a openssl config file in the nix store, and interpolate that here.